### PR TITLE
Update glossary.rst

### PR DESCRIPTION
**Pull request by Naereen**
- [X] Correct spacing (trainling spaces, and ". " → ".  " to be consistent),
- [X] And a few typing mistake ("the the" → "the", "wikipedia" → "Wikipedia"),
- [X] Add a link or two (NeuroSpin, linkify two Wikipedia URL).

### CODE REVIEW

File: doc/glossary.rst [hunk 1]
```diff
@@ -27,17 +27,17 @@
       hemoglobin.  Oxy- and deoxy- hemoglobin have different magnetic
       properties.  This in turn leads to a change in MRI signal that
       can be detected by collecting suitably sensitive MRI images at
-      regular short intervals during the blood flow chance.  See the
-      the `wikipedia FMRI`_ article for more detail.
+      regular short intervals during the blood flow chance.
```
Comment (matthew-brett) [inline doc/glossary.rst]: Aha - you fix is good - but there's another mistake I made 'chance' for 'change'.

File: doc/glossary.rst [hunk 2]
```diff
@@ -75,19 +75,19 @@
...
       The GPL_ is the GNU general public license.  It is one of the
-      most commonly-used open-source sofware licenses.  The
+      most commonly-used open-source software licenses.  The
       distinctive feature of the GPL license is that it requires that
       any code derived from GPL code also uses a GPL license.  It also
       requires that any code that is statically or dynamically linked
-      to GPL code has a GPL-compatible license. See:
-      http://en.wikipedia.org/wiki/GNU_General_Public_License and
-      http://www.gnu.org/licenses/gpl-faq.html
+      to GPL code has a GPL-compatible license.  See: `Wikipedia GPL
```
Comment (matthew-brett) [inline doc/glossary.rst]: Could these already be in the general `links_names` file included at the end of this file?

**Comment by matthew-brett** [comment]
Thanks for doing that.

**Comment by Naereen** [comment]
Any chance of merging this PR?

**Comment by matthew-brett** [comment]
Sure - very happy to do that.   I made some comments in the hope they were helpful - did you have any thoughts about those?  Or would you rather I merged and edited afterwards?

File: doc/glossary.rst (hunk 2 above)
Comment (Naereen) [inline doc/glossary.rst]: I think the link to the GPLicense is not used outside this glossary.rst document.

**Comment by Naereen** [comment]
- [X] chance → change, as you asked;
- For the few Wikipedia links, I think they are not used outside this glossary.rst document, that's why I didn't include them in `links_names.txt`.

**Comment by matthew-brett** [comment]
Thanks - it was kind of you to check those small things - merging.

**Pull request merged**